queries.sql

```diff
@@ -15,3 +15,43 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
- [ ] I would recommend selecting and verifying the `species` that change was made after update instead of selecting all the columns.

```sql
...
SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; 
```